plugins/uefi/fu-plugin-uefi.c

```diff
@@ -30,6 +30,8 @@
 #include "fu-plugin.h"
 #include "fu-plugin-vfuncs.h"
 
+#define UEFI_GUID_IMAGE_UPLOAD		"3b8c8162-188c-46a4-aec9-be43f1d65697"
```

Comment:
Can this be defined from `libfwup `instead of hardcoded here?  If you're going to have to depend on a newer version of `libfwup `anyway, I think it's better to keep that detail there in case `libfwup `ever finds a need to change it (such as a clash with another GUID on the system).

Reply:
@vathpela -- seems like a good idea to me.

Reply:
Well so it turns out that GUID is part of the ux-capsule spec, so probably not going to change ever.  @vathpela I think it would still be nice for `libfwup` to provide it in an include rather than defining in two places.

Reply:
So this isn't lost in outdated comment land, putting this here for now: https://github.com/rhboot/fwupdate/issues/82